src/main/resources/application.yml

```
@@ -37,4 +37,9 @@ management:
         include: info, health, metrics, loggers
   info:
     env:
-      enabled: true
\ No newline at end of file
+      enabled: true
+# 기상청 API 설정
+weather:
+  api:
+    serviceKey: ${WEATHER_API_KEY}
+    url: ${WEATHER_API_URL}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)